src/victory-util/add-events.js

```diff
@@ -0,0 +1,61 @@
+import { defaults, assign, isFunction, partialRight } from "lodash";
+import Events from "./events";
+
+export default (WrappedComponent) => {
+  return class addEvents extends WrappedComponent {
+    constructor() {
+      super();
+      this.state = {};
+      const getScopedEvents = Events.getScopedEvents.bind(this);
+      this.getEvents = partialRight(Events.getEvents.bind(this), getScopedEvents);
+      this.getEventState = Events.getEventState.bind(this);
+    }
+
+    componentWillMount() {
```

Comment:
If the wrapped component uses `state` it is not that unlikely that something ends up broken even if methods on both classes are called, eg. if they both write/read to the same `state` key, or if WrappedComponent clears state at some point in the lifecycle. 
But I think it's fine ( I hope) to just not support WrappedComponent to use `state`. 

about the two lifecycle methods used here, can they conceivably collide in some way, like can one class's use of the lifecycle method break something in the other? should the `super` methods be called before or after the `addEvents` ones?

Is it worth it to make this a HOC that just renders `WrappedComponent` instead of `extend`-ing it to avoid all this?